gluster-exporter/metric_ps.go

```diff
@@ -199,36 +197,36 @@ func ps() {
 		}
 
 		if err != nil {
-			// TODO: Log error
+			log.WithError(err).Error("Error getting labels")
 			continue
 		}
 
 		pcpu, err := strconv.ParseFloat(lineData[1], 64)
 		if err != nil {
-			// TODO: Log Error
+			log.WithError(err).Error("Unable to parse pcpu value")
 			continue
 		}
 
 		pmem, err := strconv.ParseFloat(lineData[2], 64)
 		if err != nil {
-			// TODO: Log Error
+			log.WithError(err).Error("Unable to parse pmem value")
 			continue
 		}
 		rsz, err := strconv.ParseFloat(lineData[3], 64)
 		if err != nil {
-			// TODO: Log Error
+			log.WithError(err).Error("Unable to parse rsz value")
 			continue
 		}
 
 		vsz, err := strconv.ParseFloat(lineData[4], 64)
 		if err != nil {
-			// TODO: Log Error
+			log.WithError(err).Error("Unable to parse vsz value")
 			continue
 		}
 
 		etimes, err := strconv.ParseFloat(lineData[5], 64)
 		if err != nil {
-			// TODO: Log Error
+			log.WithError(err).Error("Unable to parse etimes value")
```

Comment:
Include additional details as well, to understand why it is failed

```
log.WithError(err).WithFields(log.Fields{
    "value", lineData[5],
    "command": lineData[6],
    "pid": lineData[0],
}).Error("Unable to parse etimes value")
```

Reply:
Ack